etc/bento.env

```diff
@@ -346,7 +346,8 @@ BENTOV2_GOHAN_PRIVATE_AUTHZ_URL=http://${BENTOV2_GOHAN_AUTHZ_OPA_CONTAINER_NAME}
 # Bento-Public
 
 BENTO_PUBLIC_IMAGE=ghcr.io/bento-platform/bento_public
-BENTO_PUBLIC_VERSION=0.12.0
+# BENTO_PUBLIC_VERSION=0.12.0
+BENTO_PUBLIC_VERSION=pr-84
```

Comment:
Change to 'edge' when pento_public [pr](https://github.com/bento-platform/bento_public/pull/84) is merged